controllers/articles.controller.js

```diff
@@ -0,0 +1,18 @@
+//require model
+const fetchArticleById = require("../models/articles.model.js");
+
+//extract parameter and invoke model with it
+function getArticleById (req, res, next) {
+    const article_id = req.params.article_id;
+    fetchArticleById(article_id)
+    .then((result) => {
+        return res.send(result);
+    })
+    .catch((error) => {
+        next(error);
+    })
+    
+}
```

Comment:
1. Comment on line 4 isn't needed, this is more for yourself rather than the person reading the code
2. `getArticleById` is a well named function name
3. Extracting the article_id into its own variable is good, and makes this cleaner and easier to read
4. On line 9: you don't appear to be sending a response status code of `200`? Is this intentional? Yes, express will send that by default but we shouldn't rely on defaults, instead you should explicitly state the status code to respond with in all responses.
5. Also on line 9: it's convention to send data back on an object with a key/value pair, here you're just returning the single object as is, instead you should return the data on a key, such as `.send({article: result})`
6. Good work putting in the catch block and invoking next to ensure the middleware actions